### ISSUE TITLE
Could a spack package be provided for hiprand?

### ISSUE THREAD

**Issue by jmellorcrummey** [ROCmSoftwarePlatform/rocRAND#278]
**Is your feature request related to a problem? Please describe.**
There is not a download of hiprand for centos8 as a yum package. Here (https://repo.radeon.com/rocm/centos8/5.2/main), I see that there are yum packages for hipfft, hibblas and others. Could a centos package be added for hiprand?

**Describe the solution you'd like**
Could a centos package be added for hiprand?

Alternatively, the DOE favors software distribution through spack. There are already spack packages for ROCm components. 
How about adding a spack package for hiprand?

**Comment by cgmb** [comment]
I'm not a rocrand / hiprand developer so I'll leave the CentOS package question up to them, but I think @srekolam is the one to ask about the status of hiprand in Spack.

**Comment by saadrahim** [comment]
For your immediate needs, using rocRAND packages should work as hipRAND is included. I suspect the current spack build for rocRAND also builds hipRAND. But @srekolam should be able to confirm the spack status.

**Comment by srekolam** [comment]
yeah.. currently we are building hipRAND as part of rocRAND build.

**Comment by jmellorcrummey** [comment]
Thanks for the responses! I was able to 

yum install rocrand-devel

and get everything I needed. I had originally looked at the hiprand README.md file and it mentioned installing hiprand as a package using "apt". It would be nice to mention somewhere that hiprand is part of rocrand and how to install the whole thing in your README.md. 

I ended up stumbling into this because someone in my group was trying to build the AMReX package, which called for hiprand and we didn't know where to get it other than building from source. We were hoping that there was an official distribution. Now I know!

Thanks again!
